functions/src/index.ts

```diff
@@ -38,15 +22,45 @@ userApp.get('/users/:userId/', (request, response) => {
   response.redirect("/users/" + request.params.userId + "/invites" )
 })
 
-userApp.post('/users/:userId/invites', (request: RequestWithAuth, response) => {
-  if (!request.auth || !request.auth.uid) {
-    response.status(403)
-    response.send("User needs to sign in")  
-  } else if(request.params.userId != request.auth.uid) {
-    response.status(403)
-    response.send("The signed-in user is not allowed to POST a resource to the given path")  
-  } else { //request.params.userId == request.auth.uid)
+userApp.post('/users/:userId/invites', (request, response) => {
+  console.log('Check if request is authorized with Firebase ID token');
```

Comment:
From an official Firebase sample.

[Stack Overflow - How to protect firebase Cloud Function HTTP endpoint to allow only Firebase authenticated users?
](https://stackoverflow.com/questions/42751074/how-to-protect-firebase-cloud-function-http-endpoint-to-allow-only-firebase-auth)

https://github.com/firebase/functions-samples/blob/d6cf223b6703964f1e942a5e0faa907ca84afee4/authorized-https-endpoint/functions/index.js